pkgs/development/web/bun/default.nix

```diff
@@ -4,40 +4,43 @@ let
     aarch64-darwin = {
       arch = "aarch64";
       shortName = "darwin";
-      sha256 = "c82547d96125bf93ae76dafe203cae5f7cd50d041bfb1cf972f9f0232a0d1cc1";
+      sha256 = "1pi9k5bipcgay8r2vppxclb5hiy7cf0fiwsn5xgjqgdsh1mwcpj7";
     };
 
     aarch64-linux = {
       arch = "aarch64";
       shortName = "linux";
-      sha256 = "3430f3ff456ee86ddb607a46ee937c9c1a02b8e4d2546de52b4493878f66afb8";
+      sha256 = "14x31bbnm2yra0fxyy4c4r7y0q605m3q3c2dfkpd8w2jyqz8anh9";
     };
 
     x86_64-darwin = {
       arch = "x64";
       shortName = "darwin";
-      sha256 = "51fb5f29b5f00207ede11c892ccf5bb3ab437b77e7420e1c18b7fc91e02e2494";
+      sha256 = "14x31bbnm2yra0fxyy4c4r7y0q605m3q3c2dfkpd8w2jyqz8anh9";
     };
 
     x86_64-linux = {
       arch = "x64";
       shortName = "linux";
-      sha256 = "89fe00713a4e0e9f77d8842c5e07f771bd743271746fcb755c5d98cb5c00456e";
+      sha256 = "0wfn2aixk5f8z0g310nypg413lklbjr18nisivxzdhkx58zlcy1p";
     };
   };
-  dist = dists.${stdenvNoCC.hostPlatform.system} or (throw "Unsupported system: ${stdenvNoCC.hostPlatform.system}");
+  dist = dists.${stdenvNoCC.hostPlatform.system} or (throw
+    "Unsupported system: ${stdenvNoCC.hostPlatform.system}");
```

Comment:
Please revert this ...

Reply:
Mea culpa, adjusted my formatter, thanks!